R/03-geojob-obj.R

```diff
@@ -56,6 +56,31 @@ setMethod("geojob", signature(), function(...) {
   return(geojob)
 })
 
+#'@param xml location of xml 
+#'@rdname geojob-methods
+#'@importFrom httr GET 
+#'@importFrom RCurl url.exists
+#'@importFrom XML xmlTreeParse xmlAttrs xmlRoot
+#'@aliases geojob,geojob-method
+setMethod("geojob", signature("ANY"), function(xml, urlSlot = NULL) {
```

Comment:
For S4 if you want a default, you have to create a method for "missing" and set the value of `urlSlot` in that and then call the other method where that parameter isn't missing (it's "character" method)